prql-compiler/tests/integration/main.rs

```diff
@@ -184,14 +303,15 @@ fn test_rdbms() {
     })
 }
 
-fn setup_connection(con: &mut dyn DbConnection, runtime: &Runtime) {
+fn setup_connection(con: &mut DbConnection, runtime: &Runtime) {
```

Comment:
I guess this could be part of `DbConnection` (but this is me being very refine-y now, no need to change)